mitodl/mitxpro
# asadiqbal08/ Bulk purchase: update receipt page

#### Pre-Flight checklist


#### What are the relevant tickets?
[Ticket](https://trello.com/c/pldsNbPn/83-bulk-purchase-update-receipt-page)

#### What's this PR do?
Update the UI as shown in the Invision mockup in the ticket.

#### How should this be manually tested?
Visit the bulk purchase receipt page and verify the updated UI and some new added fields.

#### Screenshots (if appropriate)

#### PREVIOUS DESIGN: 
![screencapture-xpro-odl-local-8053-ecommerce-bulk-receipt-2020-03-05-16_05_47](https://user-images.githubusercontent.com/7334669/75975966-4d01fe80-5efb-11ea-9d1f-95dc466803ca.png)

#### NEW UPDATED DESIGN: 

![screencapture-xpro-odl-local-8053-ecommerce-bulk-receipt-2020-03-06-11_25_37](https://user-images.githubusercontent.com/7334669/76058096-3f9d5080-5f9d-11ea-8555-b818c4e220fd.png)

------------------------------------------------
------------------------------------------------

![screencapture-xpro-odl-local-8053-ecommerce-bulk-receipt-2020-03-06-11_25_25](https://user-images.githubusercontent.com/7334669/76058104-44620480-5f9d-11ea-8ea7-3285de74b6b6.png)
